internal/cmd/rsl/log/log.go

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package log
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct{}
+
+func (o *options) AddFlags(_ *cobra.Command) {}
+
+func (o *options) Run(_ *cobra.Command, _ []string) error {
+	repo, err := repository.LoadRepository()
+	if err != nil {
+		return err
+	}
+	entries, annotationMap, err := repository.LogRSLEntries(repo)
+	if err != nil {
+		return err
+	}
+	log := ""
+
+	for index := len(entries) - 1; index >= 0; index-- {
```

Comment:
I don't like having this much logic here, I suggest reversing the list in the repository package instead.

Reply:
I have moved this to repository/rsl. I have also added a new function in common to reverse slices of any type